presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestAggregations.java

```diff
@@ -1269,4 +1269,60 @@ public void testOrderedAggregations()
                         "('5-LOW', 445 , NULL)," +
                         "('1-URGENT', 781 , ('O'))");
     }
+
+    /**
+     * Comprehensive correctness testing is done in the TestQuantileDigestAggregationFunction
```

Comment:
Do you mean these are smoke tests? :)

Reply:
Yeah, just to verify the integration with the Presto aggregation framework.